src/FileStream.cpp

```diff
@@ -55,14 +56,14 @@ bool pocolog_cpp::FileStream::open(const char* fileName, std::ios_base::openmode
     writePos = 0;
     this->fileName = fileName;
     
-//     std::cout << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
+    LOG_DEBUG_S << "File opened " << (fd > 0) << " file size " << fileSize  << " blk size " << blockSize << std::endl;
```

Comment:
LOG_DEBUG_S comes with an std::endl at the end, so you could remove it here (holds for the other statements as well)

Reply:
I see, there are a lot of std::endl left in the code. I'm going to remove them.

Reply:
Done